src/types/actions.lua

```diff
@@ -0,0 +1,14 @@
+export type Action<Type = any> = {
```

Comment:
add reference to this other typing source: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/redux-actions/index.d.ts
```suggestion
export type Action<Type = string> = {
```

Reply:
changed to string